src/applications/mhv/medications/containers/RefillPrescriptions.jsx

```diff
@@ -152,55 +158,50 @@ const RefillPrescriptions = ({ refillList = [], isLoadingList = true }) => {
             text="Select all"
           />
           <div>
-            {fullRefillList
-              .slice()
-              .filter(rx => rx.refillRemaining > 0)
-              .map((prescription, idx) => (
-                <div key={idx} className="vads-u-margin-bottom--2">
-                  <input
-                    data-testid={`refill-prescription-checkbox-${idx}`}
-                    type="checkbox"
-                    checked={
-                      selectedRefillList.includes(
-                        prescription.prescriptionId,
-                      ) || false
-                    }
-                    id={`checkbox-${prescription.prescriptionId}`}
-                    name={prescription.prescriptionId}
-                    className="vads-u-margin-y--0"
-                    onChange={e =>
-                      e.type === 'change' &&
-                      onSelectPrescription(prescription.prescriptionId)
-                    }
-                  />
-                  <label
-                    htmlFor={`checkbox-${prescription.prescriptionId}`}
-                    className="vads-u-margin-y--0 vads-u-font-size--h4 vads-u-font-family--serif vads-u-font-weight--bold"
-                  >
-                    {prescription.prescriptionName}
-                  </label>
-                  <p
-                    className="vads-u-margin-left--4 vads-u-margin-top--0"
-                    data-testid={`refill-prescription-details-${
-                      prescription.prescriptionNumber
-                    }`}
-                  >
-                    Prescription number: {prescription.prescriptionNumber}
-                    <br />
-                    <span data-testid={`refill-last-filled-${idx}`}>
-                      Last filled on{' '}
-                      {dateFormat(
-                        prescription.rxRfRecords?.[0]?.[1]?.find(
-                          record => record.dispensedDate,
-                        )?.dispensedDate || prescription.dispensedDate,
-                        'MMMM D, YYYY',
-                      )}
-                    </span>
-                    <br />
-                    {prescription.refillRemaining} refills left
-                  </p>
-                </div>
-              ))}
+            {fullRefillList.slice().map((prescription, idx) => (
+              <div key={idx} className="vads-u-margin-bottom--2">
+                <input
```

Comment:
Is it possible to use the design system web component for this checkbox?

- https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox--default
- https://design.va.gov/storybook/?path=/docs/uswds-va-checkbox-group--default

Reply:
@jamigibbs As far as I know, the VA design system checkbox doesn't allow bold text for the label. I tried to get around the shadow-dom in a few different ways but wasn't able to. For this we're showing details below each checkbox and we need the text to be bold according to the mocks I was given from UCD.

Reply:
@agravell047 Once you have the ticket created about adding `part` to the label of the web component, can you link to it here? [Slack convo for reference](https://dsva.slack.com/archives/CBU0KDSB1/p1710940678265029).